js/weechat.js

```diff
@@ -290,7 +290,7 @@
                     var ret = {};
                     var optionCode = parseInt(m[1]);
 
-                    if (optionCode > 43) {
```

Comment:
why was this 43? seems an arbitrary number, and was smaller than the size of the color array